js/models/conversations.js

```diff
@@ -72,6 +72,7 @@
         this.messageCollection.on('send-error', this.onMessageError, this);
 
         this.on('change:avatar', this.updateAvatarUrl);
+        this.on('change:profileAvatar', this.updateAvatarUrl);
```

Comment:
I think we need a `change:profileKey` handler here which will go fetch the user's profile, thereby pulling new profile information. I just tried sharing my profile key via a message, and because the recipient already had the conversation open, nothing changed. Left the conversation and came back, still no change, because we throttle profile get. Had to restart to see the profile avatar.

Reply:
done